jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -64,37 +71,41 @@ public void render(Context context,
         } else {
             tooltipClasses.remove(ERROR_PRESENT_STYLE);
         }
-
-        if (viewPresenter.getViewErrorsActionCondition().test(value)) {
+        
+        if (viewPresenter.isUserAuthorizedForPerspective(EXECUTION_ERRORS) && errCount > 0) {
```

Comment:
@mcivantos-tribalyte can we keep the previous format with a condition.test to evaluate this?